x/docs/md/intro/getting-started.md

```diff
@@ -3,7 +3,7 @@
 1. Download and configure the [command-line client](/cli).
 1. Fetch the first exercise (`exercism fetch TRACK_ID`).
 1. Write a solution to make the test suite pass.
-1. Submit your solution (`exercism submit path/to/file`).
+1. Submit your solution (`exercism submit path/to/file` or `exercism s path/to/file{.h,.cpp}`). 
```

Comment:
Nice to know, I just never considered doing so.

So you are on Mac, what shell?  That also may be nice to mention for the PR?  This may answer the "Does it Windows" question in itself.